app/helpers/layout_helper.rb

```diff
@@ -5,7 +5,10 @@ def title(page_title, page_header = nil)
   end
 
   def title_actions(*elements)
-    content_for(:title_actions) { elements.join(" ").html_safe }
+    content_for(:title_actions) do
+      pagelets = Pagelets::Manager.sorted_pagelets_at(params, :title_action_buttons).collect { |pt| render(pt.partial.chomp).chomp }
```

Comment:
We have a good practice to end ERB files with a newline, this removes it. I wanted to prevent the extra newline which renders to additional space. I should perhaps keep just one.